dcc-portal-ui/app/scripts/survivalanalysis/js/css-to-include-in-svg-export.txt

```diff
@@ -0,0 +1,62 @@
+.domain {
```

Comment:
Managed to reuse the existing scss file

Tried to do that earlier with no luck due to interference from existing loaders, dug into the webpack documentation this time and found

> It's possible to overwrite any loaders in the configuration by prefixing the entire rule with !.

So yay, reuse